apis/i18n.d.ts

```diff
@@ -0,0 +1,46 @@
+import { csn, entity } from './cds'
+
+
+declare class I18nBundle {
+  constructor(options: I18nFilesOptions)
+  for(key: number | string | object, locale?: string | object, args?: object): string | undefined
+  at(key: number | string | object, locale?: string | object, args?: object): string | undefined
+  file: string
+  files: I18nFiles
+  get defaults(): Record<string, string>
+  get fallback(): Record<string, string>
```

Comment:
why are these two getters instead of just regular properties?

Reply:
These are defined as getter in `/lib/i18n/bundles.js`.